docs/core/diagnostics/dotnet-trace.md

```diff
@@ -5,37 +5,44 @@ author: sdmaclea
 ms.author: stmaclea
 ms.date: 10/14/2019
 ---
-# Trace for performance analysis utility (`dotnet-trace`)
+# dotnet-trace performance analysis utility
 
 **This article applies to:** .NET Core 3.0 SDK and later versions
 
-## Installing `dotnet-trace`
+## Install dotnet-trace
 
-To install the latest release version of the `dotnet-trace` [NuGet package](https://www.nuget.org/packages/dotnet-trace), use the [dotnet tool install](../tools/dotnet-tool-install.md) command:
+Install `dotnet-trace` [NuGet package](https://www.nuget.org/packages/dotnet-trace) with the [dotnet tool install](../tools/dotnet-tool-install.md) command:
 
 ```dotnetcli
 dotnet tool install --global dotnet-trace
 ```
 
 ## Synopsis
 
-```console
+```dotnetcli
 dotnet-trace [-h, --help] [--version] <command>
 ```
```

Comment:
I don't feel that this command is really a .NET CLI command.

Reply:
Our preferred syntax is `dotnet-trace`.  It could also be written as `dotnet trace` where it is truly a .NET CLI command. I have chosen not to advertise the second syntax.